src/documentation/dsls/sql/shell.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: section
+title: "Beam SQL: Shell"
+section_menu: section-menu/sdks.html
+permalink: /documentation/dsls/sql/shell/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Beam SQL Shell
+
+## Overview
+
+Starting with version 2.6.0, Beam SQL includes an interactive shell, called the Beam SQL shell. The shell allows you to write pipelines as SQL queries without needing the Java SDK. By default, Beam uses the `DirectRunner` to execute the queries as Beam pipelines.
+
+This page describes how to work with the shell, but does not focus on specific features of Beam SQL. For a more thorough overview of the features used in the examples on this page, see the corresponding sections in the [Beam SQL documentation](https://beam.apache.org/documentation/dsls/sql/overview/).
+
+
+## Quickstart
+
+To use Beam SQL shell, you must first clone the [Beam SDK repository](https://github.com/apache/beam). Then, from the root of the repository clone, execute the Gradle task to run the shell ([source code](https://github.com/apache/beam/blob/master/sdks/java/extensions/sql/shell/build.gradle#L39)) by running the following command:
```

Comment:
Linking to line numbers in master is a bad idea. Can you use a release tag or commit?

Reply:
That will become stale quick as well. Do you think linking just to the file is better?

Reply:
Just linking to the file would probably be good. The target audience of the SQL shell probably doesn't care about the code, so just dropping the links might be a good option too.